### ISSUE TITLE
[About]: User is unable to assign any shortcut to activate 'About' button.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15986]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Navigate to the 'About' and observe the issue.

**User Experience: **
All users UX will enhance if user able to activate control from its editing area by just pressing its shortcut key as currently users must navigate to that specific control to activate it.

**Attachment:**
[User is unable to assign any shortcut to activate 'About' button..zip](https://github.com/microsoft/terminal/files/12644866/User.is.unable.to.assign.any.shortcut.to.activate.About.button.zip)

### Expected Behavior

User should be able to assign any shortcut to activate 'About' button.

### Actual Behavior

User is unable to assign any shortcut to activate 'About' button.
